examples/next/pages/ui/components/in-app-messaging/demo/utils.ts

```diff
@@ -19,6 +20,7 @@ export interface GetDemoMessageParams {
   primaryButtonAction: InAppMessageAction;
   secondaryButtonAction: InAppMessageAction;
   layout: InAppMessageLayout;
+  useAnalyticsActions: boolean;
```

Comment:
Can we replace the usage of "actions" in this PR with "events"? Just thinking that is more aligned with the component and its usage

Reply:
Updated to useAnalyticEvents